src/components/media-player/media-player-controls.js

```diff
@@ -96,16 +80,28 @@ function MediaPlayerControls({
       </div>
       <div className="MediaPlayerControls-controlsGroup">
         <div>
-          {playbackRate !== NORMAL_PLAYBACK_RATE && (
-            <span className="MediaPlayerControls-playbackRate">x {formatNumber(playbackRate)}</span>
-          )}
           <Dropdown
+            placement="top"
             trigger={['click']}
-            placement="bottomRight"
-            menu={{ items: getSettingsMenuItems(), onClick: handleSettingsMenuItemClick }}
+            menu={{ items: getPlaybackRateMenuItems(), onClick: handlePlaybackRateMenuItemClick }}
             >
-            <Button type="link" icon={<SettingsIcon />} />
+            <Button
+              type="link"
+              icon={playbackRate === DEFAULT_MEDIA_PLAYBACK_RATE ? <BrandSpeedtestIcon /> : null}
+              >
+              {playbackRate === DEFAULT_MEDIA_PLAYBACK_RATE ? null : renderMediaPlaybackRate()}
+            </Button>
           </Dropdown>
+          {!!onLoopMediaChange && (
+            <Button
+              type="link"
+              icon={loopMedia ? <RepeatIcon /> : <RepeatOffIcon size="1em" stroke={1} />}
```

Comment:
If this is the only usage of `RepeatOffIcon` I think we can safely put the size and stroke inside, right?

Reply:
It is actually hard-coded to `1em` and `1.5` for each of the tabler icons, these props didn't have any effect, I forgot to remove them. Thanks for pointing it out! They're gone now.